src/stores/deck.js

```diff
@@ -41,7 +41,7 @@ const useDeckStore = create((set, get) => ({
   },
   addCardToDeck: card =>
     set(state => ({
-      deck: [...state.deck, card],
+      deck: shuffleDeck({ ...state.deck, card }),
```

Comment:
i think this method takes an array, not an object. 

https://github.com/garretthogan/blackjack-frontend/pull/16/commits/a357e257f5ed1bd3bc767271f00f36c0ba07eb3b#diff-72b928e8286988c9c5264b3bb3bbb84226f130856ead35fdf6abeab00e9cc398R17